pkg/exchange/binance/exchange.go

```diff
@@ -150,6 +174,14 @@ func New(key, secret string) *Exchange {
 	return ex
 }
 
+func (e *Exchange) GetEd25519PrivateKey() ed25519.PrivateKey {
+	return e.ed25519PrivateKey
+}
+
+func (e *Exchange) GetEd25519Auth() bool {
```

Comment:
ditto, it's for the stream, they are in the same package, so you don't have to export them as public methods, it could be private methods